templates/2_Pixie_Cluster_Pixels.ipynb

```diff
@@ -31,16 +31,42 @@
     "from ark.utils import data_utils, example_dataset, plot_utils\n",
```

Comment:
<p>To be consistent with the way this path is specified for the example dataset above, can we change<code>{base_dir}/image_data</code>  to <code>../data/example_dataset/image_data</code> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1063/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>